WikiPreview.Fluent.Plugin/WikiPreviewSearchApp.cs

```diff
@@ -108,59 +109,88 @@ public ValueTask LoadSearchApplicationAsync()
             {
                 if (task.IsCompletedSuccessfully)
                     _ = task.Result?.Query.Pages.ParallelForEachAsync(async entry =>
-                    {
-                        (_, PageView value) = entry;
-                        string resultName = value.Extract;
-                        string displayedName = value.Title;
-                        double score = displayedName.SearchDistanceScore(searchedText);
-                        string pageId = value.PageId.ToString();
-                        string wikiUrl = WikiRootUrl + displayedName;
-                        BitmapImageResult bitmapImageResult;
-
-                        string additionalInfo = "";
-                        if (!string.IsNullOrWhiteSpace(resultName))
                         {
-                            using var reader = new StringReader(resultName);
-                            additionalInfo = await reader.ReadLineAsync().ConfigureAwait(false) ?? resultName;
-                        }
-
-                        if (value.Thumbnail != null)
-                        {
-                            string imgUrl = value.Thumbnail.Source;
-                            using var imageClient = new HttpClient();
-                            Stream stream = await imageClient.GetStreamAsync(imgUrl, cancellationToken)
+                            WikiPreviewSearchResult wikiPreviewSearchResult =
+                                await GenerateSearchResult(entry.Value, searchedText).ConfigureAwait(false);
```

Comment:
Just for your info, the context inside new tasks is the default one, so no really need to use `ConfigureAwait(false)`